sed/binning.py

```diff
@@ -624,14 +636,16 @@ def numba_histogramdd(
         hist = _hist_from_bins(
             sample,
             tuple(bins),
-            tuple(b.size for b in bins),
+            tuple(b.size - 1 for b in bins),
         )
         return hist, bins
 
     elif method == "int":
         # normalize the range argument
         if ranges is None:
             ranges = (None,) * D
```

Comment:
What good does this do? For which case is this intended?

Reply:
Actually I cannot recall, I think it still remains from the MPES implementation. But it should never happen that ranges is None in the "int" method, so I will remove it

Reply:
actually, even though it might be redundant, I will add an error here in case this happens, as it should not.